scraping/x/adajobo_scrapper.py

```diff
@@ -0,0 +1,372 @@
+import asyncio
+import threading
+import traceback
+import bittensor as bt
+from typing import List
+from common import constants
+from common.data import DataEntity, DataLabel, DataSource
+from common.date_range import DateRange
+from scraping.scraper import ScrapeConfig, Scraper, ValidationResult
+from scraping.apify import ActorRunner, RunConfig
+from scraping.x.model import XContent
+from scraping.x import utils
+import datetime as dt
+
+
+class AdajoboTwitterScraper(Scraper):
```

Comment:
nit: I think the actor is called "ApiDojo": https://console.apify.com/actors/61RPP7dywgiy0JPD0/input

Reply:
Yes you are correct.